back-end/routes/chain/search_transactions.js

```diff
@@ -19,6 +27,29 @@ module.exports = async function(req, res) {
     res.send(retObj);
 };
 
+async function checkAuthorized(accountName, permission, pubKey, signature, signatureDate) {
+    const blockchainAccount = await accountability.getAccount(accountName);
+
+    if (!blockchainAccount) {
+        throw new Error('Authorizing account not found');
+    }
+
+    const blockchainActivePermission = blockchainAccount.permissions.find(element => element.perm_name === permission);
+
+    if (!blockchainActivePermission) {
+        throw new Error('Authorizing account permission not found');
+    }
+
+    const signDate = new Date(signatureDate);
+    const now = new Date();
+    if (now.getSeconds() - signDate.getSeconds() > 15) {
```

Comment:
It seems you want to expire the validity of the signature after 15 seconds.
getSeconds will return the seconds from 00 to 59
If I am not mistaken, you might want to use `getTime() / 1000` to get Unix epoch seconds